spec/cc/analyzer/formatters/html_formatter_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+
+module CC::Analyzer::Formatters
+  describe HTMLFormatter do
+    include Factory
+
+    let(:formatter) do
+      filesystem ||= CC::Analyzer::Filesystem.new(
```

Comment:
FWIW the return value of `let`s are memoized by rspec, so I don't think the `||` is doing anything for you here?